### PR TITLE
[NET] Map Reading Endpoint

### PR DESCRIPTION
**Pull request by OchiengPaul442**
#### Summary of Changes (What does this PR do?)

- **Endpoint Update**: Transitioned from the deprecated events endpoint to the newly implemented Map reading endpoint, enhancing the application's data retrieval process.
- **Enhanced Error Handling**: Incorporated additional error handling mechanisms during Client and token generation
- **Optimization**:  Enhanced the OverlayMap code to improve performance, ensuring a seamless experience for first-time users on the platform. This optimization addresses and resolves the issue of missing data on nodes due to initial load challenges on the map. As a result, the map now loads more efficiently and provides complete data from the outset

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state

#### Screenshots (optional)
image

### CODE REVIEW

File: netmanager/src/views/pages/Map/OverlayMap.js [hunk 1]
```diff
@@ -655,21 +640,20 @@ const getStoredData = () => {
   return { storedData, storedTimeStamp };
 };
 
+const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
```
Comment (Baalmart) [inline netmanager/src/views/pages/Map/OverlayMap.js:643]: thanks @OchiengPaul442 , please share more insights about what this does?

File: netmanager/src/views/pages/Map/OverlayMap.js [hunk 2]
```diff
@@ -388,15 +385,20 @@ export const OverlayMap = ({ center, zoom, heatMapData, monitoringSiteData }) =>
   const [showSensors, setShowSensors] = useState(true);
   const [showCalibratedValues, setShowCalibratedValues] = useState(false);
   const [showHeatMap, setShowHeatMap] = useState(false);
+  const mapContainerRef = useRef(null);
   const initialShowPollutant = {
-    pm2_5: handleLocalStorage.get('pollutant') === 'pm2_5',
-    no2: handleLocalStorage.get('pollutant') === 'no2',
-    pm10: handleLocalStorage.get('pollutant') === 'pm10'
+    pm2_5: false,
```
Comment (Baalmart) [inline netmanager/src/views/pages/Map/OverlayMap.js:390]: @OchiengPaul442 , also for this section, what is happening when these pollutants are assigned to false?

**Review by Baalmart — COMMENTED** [review]
thanks @OchiengPaul442 ,just left a few code comments for your review and feedback. I will also run the PR to test out the changes in UI.

**Review by Baalmart — APPROVED** [review]
thanks @OchiengPaul442 , let me merge and we monitor from staging environment.

File: netmanager/src/views/pages/Map/OverlayMap.js (hunk 1 above)
Comment (OchiengPaul442) [reply netmanager/src/views/pages/Map/OverlayMap.js:643]: @Baalmart, the code just calculates the 24hr time in milliseconds. It's then used to update the data local storage only when 24hr have elapsed.

**Pull request merged**